Sources/MongoSwift/Operations/ListIndexesOperation.swift

```diff
@@ -9,21 +9,35 @@ internal enum ListIndexesResults {
     case names([String])
 }
 
+/// Options to use when creating listing indexes on a `MongoCollection`.
+public struct ListIndexOptions: Encodable {
+    /// An arbitrary BSON type to help trace the operation through
+    /// the database profiler, currentOp and logs. The default is to not send a value.
```

Comment:
also need a test for this one

Reply:
Added test testCreateListDropIndexesComment() in MongoCollection+IndexTests